components/ActivityBarChart.tsx

```
@@ -19,19 +20,51 @@ const BAR_WIDTH = 28;
+    //     { day: 'Tue', value: 45, gradient: gradients.greenLinear },
+    //     { day: 'Wed', value: 75, gradient: gradients.button },
+    //     { day: 'Thu', value: 95, gradient: gradients.greenLinear },
+    //     { day: 'Fri', value: 50, gradient: gradients.button },
+    //     { day: 'Sat', value: 80, gradient: gradients.greenLinear },
+    // ];
+
+    // const maxValue = 100;
+    const [weeklyData, setWeeklyData] = useState<any[]>([]);
+
+
+    useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on loadWeeklyData.

This dependency is being used here, but is not specified in the hook dependency list.

Either include it or remove the dependency array.
Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)